docs/included.md

```diff
@@ -117,6 +117,26 @@
   - [API](api.md#scores.continuous.rmse)
   - [Tutorial](project:./tutorials/Root_Mean_Squared_Error.md)
   - [Wikipedia](https://en.wikipedia.org/wiki/Root-mean-square_deviation)
+* - Threshold Weighted Absolute Error
+  - [API](api.md#scores.continuous.tw_absolute_error)
+  - [Tutorial](project:./tutorials/Threshold_Weighted_Scores.md)
+  - [Taggart (2022)](https://doi.org/10.1002/qj.4206)
+* - Threshold Weighted Expectile Score
```

Comment:
It seems like the "*" has a space before it that shouldn't be there. I brought it in line with the others in my checkout and that enabled the docs to build.